packages/html-reporter/src/testCaseView.tsx

```diff
@@ -77,8 +68,8 @@ export const TestCaseView: React.FC<{
       {test && !!test.projectName && <ProjectLink projectNames={projectNames} projectName={test.projectName}></ProjectLink>}
       {labels && <LabelsLinkView labels={labels} />}
     </div>}
-    {!!visibleAnnotations.length && <AutoChip header='Annotations' dataTestId='test-case-annotations'>
-      {visibleAnnotations.map((annotation, index) => <TestCaseAnnotationView key={index} annotation={annotation} />)}
+    {test?.results.length === 0 && <AutoChip header='Annotations' dataTestId='test-case-annotations'>
```

Comment:
Shouldn't this check visible annotations count as well (those not starting with `_`)? Otherwise we'll have an empty section.

Reply:
good catch!